integration-tests/batchBundle.test.ts

```diff
@@ -0,0 +1,221 @@
+                            given: ['Peter', 'James'],
+                        },
+                    ],
+                    gender: 'male',
+                    birthDate: '1974-12-25',
+                },
+                request: {
+                    method: 'POST',
+                    url: '/Patient/',
+                },
+            },
+            {
+                resource: {
+                    id: 'resourceToDelete',
+                    resourceType: 'Patient',
+                    text: {
+                        status: 'generated',
+                        div: '<div xmlns="http://www.w3.org/1999/xhtml">Some narrative</div>',
+                    },
+                    active: true,
+                    name: [
+                        {
+                            use: 'official',
+                            family: 'Chalmers',
+                            given: ['Peter', 'James'],
+                        },
+                    ],
+                    gender: 'male',
+                    birthDate: '1974-12-25',
+                },
+                request: {
+                    method: 'POST',
+                    url: '/Patient/',
+                },
+            },
+            {
+                resource: {
+                    id: 'resourceToGet',
+                    resourceType: 'Patient',
+                    text: {
+                        status: 'generated',
+                        div: '<div xmlns="http://www.w3.org/1999/xhtml">Some narrative</div>',
+                    },
+                    active: true,
+                    name: [
+                        {
+                            use: 'official',
+                            family: 'Chalmers',
+                            given: ['Peter', 'James'],
+                        },
+                    ],
+                    gender: 'male',
+                    birthDate: '1974-12-25',
+                },
+                request: {
+                    method: 'POST',
+                    url: '/Patient/',
+                },
+            },
+        ],
+    };
+
+    test('post multiple batches with failures', async () => {
+        const response = await client.post('/', firstBatch);
+        expect(response.data).toMatchObject({
+            resourceType: 'Bundle',
+            type: 'batch-response',
+            entry: [
+                {
+                    response: {
+                        status: '403 Forbidden',
+                        location: 'Patient/someRandomResource',
+                        etag: '',
+                    },
+                },
+                {
+                    response: {
+                        status: '404 Not Found',
+                        location: 'Patient/someResource',
+                        etag: '',
+                    },
+                },
+                {
+                    response: {
+                        status: '201 Created',
+                        etag: '1',
```

Comment:
We actually can't do this here since the uuids are generated from the operation, despite the id field being present in the POST request. This behavior aligns with the creation of resources through normal POST requests as well, they still have a generated uuid.